pkg/gotohelm/internal/bootstrap/bootstrap.go

```diff
@@ -224,7 +224,7 @@ func resource_MustParse(repr any) any {
 	idx := -1
 	for i, s := range scales {
 		if float64(s) == float64(scale) {
-			// Ideally this would be an early return but https://github.com/redpanda-data/helm-charts/issues/1331
+			// Ideally this would be an early return but https://github.com/redpanda-data/redpanda-operator/issues/1331
```

Comment:
This need to be roll backed. 
```suggestion
			// Ideally this would be an early return but https://github.com/redpanda-data/helm-charts/issues/1331
```